eng/common/scripts/git-diff-changes.ps1

```diff
@@ -1,36 +1,49 @@
 <#
   .SYNOPSIS
   Returns git diff changes in pull request.
-
   .DESCRIPTION
   The script is to return diff changes in pull request.
-
   .PARAMETER SourceCommittish
   The branch committish PR merges from.
   Definition of committish: https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefcommit-ishacommit-ishalsocommittish
-
   .PARAMETER TargetCommittish
   The branch committish PR targets to merge into.
+  .PARAMETER IncludeRegex
+  The regex of included files.
+  .PARAMETER ExcludeMode
+  The filter type A(a)dd, D(d)elete, R(r)ename, U(u)pate. 
+  E.g. 'ad' means filter out the newly added file and deleted file 
+  E.g. '' means no filter on file mode.
+  .PARAMETER FilterType
 #>
 [CmdletBinding()]
```

Comment:
We should think about making this a helper function to make the calling a little easier in all the PS scripts. We can still have this wrapper script so we can call it as a script in some of the locations if needed.

Reply:
Yes, template uses script conveniently. Script uses helper function easily. I am thinking to have both.